videos/what-is-ap/script.md

```diff
@@ -0,0 +1,129 @@
+---
+size: 720p
+background: bach_cello.mp3 fade-out
+transition: crossfade 0.2
+theme: light
+voice: Elisabeth 
+---
+
+![](moj_logo.png)
+
+(pause: 3)
+
+This short video introduces the Ministry of Justice's Analytical Platform.
+
+(pause: 1)
+
+The Analytical Platform is a suite of browser based tools for ingesting, storing, analysing and presenting data.
+As an analyst, this is the tool you'll use most often.
+
+(pause: 1)
+
+It's developed, supported and operated by the Ministry of Justice's Analytical Platform team, who produced this video.
+
+---
+
+![](signup_github.mp4)
+
+(voice: william)
+
+(pause: 1)
+
+To use the platform sign up for GitHub, which is also where you'll store all your code.
+Once set up, email the Analytical Platform team to enable your GitHub account on the platform.
+
+(pause: 1)
+(voice: carol) 
+
+Full details of the sign-up process can be found in the links associated with this video.
+
+---
+
+![](tools_page.mp4)
+
+(voice: elisabeth)
+
+The platform provides open-source analytical software such as `R` Studio and JupiterLab.
+> The platform provides open-source analytical software such as `R` Studio and JupyterLab.
+
+(pause: 1)
+
+All your work created with such tools is stored in GitHub.
+
+---
+
+![](data_sources.mp4)
+
+(voice: william)
+
+(pause: 1)
+
+It's also possible to use cloud data services such as Amazon Athena, Glue and Redshift.
+
+Your data is usually stored in Amazon `S` 3 buckets.
+
+(pause: 1)
+
```

Comment:
In here I'd like to say "Many of the main operational databases are already available on the Analytical Platform, provided centrally by the data engineering team. This includes the main databases for prisons, probation, and criminal courts, as well as others. Please contact the data engineering team who will be happy to advise".

Then probably (as a new thought) something along the lines of "analysts are also free to analyse their own data on the Analytical Platform, subject to the usual data protection considerations".